introduction.md

```diff
@@ -0,0 +1,2 @@
+On my way to becoming a GitHub Master.  I love Ruby and JavaScript.

```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.